recipe/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "isotree" %}
+{% set version = "0.5.24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d5cb06ac442f3708aa00221f917cb5d18803c9ec256b5224a1cd08935227c9ed
+  patches:
+    - patches/0001-disable-xoshiro.patch
+    - patches/0002-add-unvendored-tsl_robin_map.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+  missing_dso_whitelist:
+    - '*/libc++.1.dylib'  # [osx]
+    - '*/libomp.dylib'    # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - intel-openmp  # [not osx and blas_impl == 'mkl']
+    - llvm-openmp   # [osx]
```

Comment:
feels like some variation is not covered here. is it relevant?

Reply:
The linux part is implicitly added through the inclusion of gcc.

Reply:
We could add an explicit `libgomp  # [linux]` @lorepirri .

Reply:
done